recipe/session/sessionwithjwt/sessionContainer.go

```diff
@@ -21,81 +21,95 @@ import (
+			// not be alive for too long (since it's expired already). So we set it to
+			// 1 second lifetime.
+			jwtExpiry = 1
+		}
+
+		newAccessTokenPayload, err = addJWTToAccessTokenPayload(newAccessTokenPayload, jwtExpiry, originalSessionClass.GetUserIDWithContext(userContext), jwtPropertyName.(string), openidRecipeImplementation, userContext)
+		if err != nil {
+			return err
+		}
+
+		return originalSessionClass.UpdateAccessTokenPayloadWithContext(newAccessTokenPayload, userContext)
+	}
 	return sessmodels.SessionContainer{
-		RevokeSession: func() error {
-			return originalSessionClass.RevokeSession()
+		RevokeSessionWithContext: func(userContext supertokens.UserContext) error {
+			return originalSessionClass.RevokeSessionWithContext(userContext)
 		},
-
-		GetSessionData: func() (map[string]interface{}, error) {
-			return originalSessionClass.GetSessionData()
+		GetSessionDataWithContext: func(userContext supertokens.UserContext) (map[string]interface{}, error) {
+			return originalSessionClass.GetSessionDataWithContext(userContext)
 		},
-
-		UpdateSessionData: func(newSessionData map[string]interface{}) error {
-			return originalSessionClass.UpdateSessionData(newSessionData)
+		UpdateSessionDataWithContext: func(newSessionData map[string]interface{}, userContext supertokens.UserContext) error {
+			return originalSessionClass.UpdateSessionDataWithContext(newSessionData, userContext)
+		},
+		GetUserIDWithContext: func(userContext supertokens.UserContext) string {
+			return originalSessionClass.GetUserIDWithContext(userContext)
 		},
-		GetUserID: func() string {
-			return originalSessionClass.GetUserID()
+		GetAccessTokenPayloadWithContext: func(userContext supertokens.UserContext) map[string]interface{} {
+			return originalSessionClass.GetAccessTokenPayloadWithContext(userContext)
 		},
-		GetAccessTokenPayload: func() map[string]interface{} {
-			return originalSessionClass.GetAccessTokenPayload()
+		GetHandleWithContext: func(userContext supertokens.UserContext) string {
+			return originalSessionClass.GetHandleWithContext(userContext)
 		},
-		GetHandle: func() string {
-			return originalSessionClass.GetHandle()
+		GetAccessTokenWithContext: func(userContext supertokens.UserContext) string {
+			return originalSessionClass.GetAccessTokenWithContext(userContext)
 		},
-		GetAccessToken: func() string {
-			return originalSessionClass.GetAccessToken()
+		GetTimeCreatedWithContext: func(userContext supertokens.UserContext) (uint64, error) {
+			return originalSessionClass.GetTimeCreatedWithContext(userContext)
 		},
-		GetTimeCreated: func() (uint64, error) {
-			return originalSessionClass.GetTimeCreated()
+		GetExpiryWithContext: func(userContext supertokens.UserContext) (uint64, error) {
+			return originalSessionClass.GetExpiryWithContext(userContext)
```

Comment:
You don't need a function body for these either.. since they are just calling the original implementation..